lib/public/views/Tags/Details/index.js

```diff
@@ -60,51 +62,49 @@ const tagDetails = (model) => {
         return [
             h('.flex-column.w-100', [
                 h('.w-100.flex-row', [
-                    h('h2.mv2.w-40', {
-                        onremove: () => {
-                            model.tags.isEditModeEnabled = false;
-                            model.tags.tagChanges = {};
-                            model.tags.clearTag();
-                        },
-                    }, `Tag: ${data.payload.text}`),
-                    h('.w-60.flex-row.items-center', {
+                    h('.flex-row.items-center.gc3', [
+                        h(
+                            'h2.mv2.flex-row',
+                            {
+                                onremove: () => {
+                                    model.tags.clearTagChanges();
+                                    model.tags.clearTag();
+                                },
+                            },
+                            `Tag: ${tag.text}`,
+                        ),
+                        tag.archived ? h('.f4.badge.bg-warning.white', 'Archived') : null,
```

Comment:
I have seen that for the `tag-overview` page the color `grey` is used for displaying the badge, while here it is used `orange` from `warning` but both badges display the same information. 
I think it would be better to use the same color `grey` to be consistent

Reply:
Done